.github/workflows/static_checks.yml

```diff
@@ -78,17 +78,28 @@ jobs:
       # For more information on shellcheck failures:
       # https://github.com/koalaman/shellcheck/wiki/Checks
       run: |
-        find ./bin -type f \
-        ! -name docker-build-proxy \
-        ! -name _docker.sh \
-        ! -name fmt \
-        ! -name lint \
-        ! -name _log.sh \
-        ! -name minikube-start-hyperv.bat \
-        ! -name _tag.sh \
-        ! -name test-cleanup \
-        ! -name _test-run.sh \
-        ! -name update-go-deps-shas \
-        ! -name *.nuspec \
-        ! -name *.ps1 \
-        | xargs -I {} bin/shellcheck -x -P ./bin {}
+        bin/shellcheck -x -P ./bin $(find . -type f \
+        ! -wholename ./bin/docker-build-proxy \
+        ! -wholename ./bin/fmt \
+        ! -wholename ./bin/lint \
+        ! -wholename ./bin/_log.sh \
+        ! -wholename ./bin/minikube-start-hyperv.bat \
+        ! -wholename ./bin/_tag.sh \
+        ! -wholename ./bin/test-cleanup \
+        ! -wholename ./bin/_test-run.sh \
+        ! -wholename ./bin/update-go-deps-shas \
+        ! -wholename ./cni-plugin/deployment/scripts/install-cni.sh \
+        ! -wholename ./.git/hooks/applypatch-msg.sample \
+        ! -wholename ./.git/hooks/commit-msg.sample \
```

Comment:
Consider excluding the pattern `./.git/hooks/*.sample`?

Reply:
> Consider excluding the pattern `./.git/hooks/*.sample`?

Yes, I was thinking about that but thought I'd put it up like this to see what your comments on this would be. Are the samples used? Or are they to be used? Regardless of what, should they be in this state or should they be fixed up? It's more a strategic and philosophical question IMO, for you guys to decide.

If you'd like me to add that exclusion pattern I will fix that, no problem.

Reply:
I assume that all of these files are part of some git infrastructure. We've never touched them:
```
:; git log .git/hooks
:; git rm -f .git/hooks/*.sample
fatal: pathspec '.git/hooks/applypatch-msg.sample' did not match any files
```

Reply:
Ok, let's go for excluding the pattern `./.git/hooks/*.sample` then?

Reply:
@joakimr-axis Yeah, I think that's best. Thanks!

Reply:
Done